eclipse-collections-code-generator/src/main/resources/impl/bag/mutable/synchronizedPrimitiveBag.stg

```diff
@@ -169,6 +171,14 @@ public class Synchronized<name>Bag
         }
     }
 
+    @Override
+    public Mutable<name>Set selectUnique()
```

Comment:
Since this is synchronized, you need to synchronize it on the lock. Also, you can delegate it to the underlying Bag.

```
synchronized (this.getLock())
{
    Mutable<name>Set result = this.getMutable<name>Bag().selectUnique();
}
return result;
```